upgrading/automated_upgrades.adoc

```diff
@@ -916,6 +933,44 @@ logging deployment.
 
 If the xref:upgrading-efk-logging-stack-script[script above] indicates your indices do not contain fields with a dot in the name, use the following steps to upgrade.
 
+. Review how to
```

Comment:
I copied these two steps from the following procedure. Not sure why I didn't put them here originally. Review as desired.

Reply:
I'm fine with these as they are since they are copied from existing.